ci/scripts/github/conda.sh

```diff
@@ -39,4 +40,15 @@ conda info
 rapids-logger "Building Conda Package"
 
 # Run the conda build and upload
-${MRC_ROOT}/ci/conda/recipes/run_conda_build.sh "$@"
+${MRC_ROOT}/ci/conda/recipes/run_conda_build.sh "${CI_SCRIPT_ARGS}"
+
+if [["${CI_SCRIPT_ARGS}" =~ "upload" ]]; then
```

Comment:
Remark: This will produce weird behavior if the script args contains something containing upload but not actually just upload, such as `skip-upload`.